README.md

```diff
@@ -150,9 +153,6 @@ There are also a few Linux distribution specific settings:
 # - elementary OS
 #######################################
 
-# Set to "false" if package cache should not be updated
-wireguard_ubuntu_update_cache: "true"
```

Comment:
I actually want to keep that variable for backward compatibility at least for now. So this could be a possible solution:
```
# Set to "false" if package cache should not be updated.
# DEPRECATED: This variable will be removed with the next major release. Please use "wireguard_update_cache" instead.
wireguard_ubuntu_update_cache: "{{ wireguard_update_cache }}"
```